x/emissions/module/rewards/scores.go

```diff
@@ -209,6 +213,7 @@ func GenerateInferenceScores(
 	}
 
 	types.EmitNewInfererScoresSetEvent(ctx, newScores)
+	types.EmitNewActorEMAScoresSetEvent(ctx, types.ActorType_ACTOR_TYPE_INFERER_UNSPECIFIED, emaScores, activeArr)
```

Comment:
You did this for all active set calculations, but forgot to for all passive set calculations.
Will do in my commit.